src/components/contacComponents/contactInfo/contactInfo.css

```diff
@@ -50,5 +56,18 @@
     position: relative;
 }
 
+@media screen and (max-width: 960px) {
+  .left{
+font-size: 14px;  }
```

Comment:
the font gets a little too small to read below 960px. I would just leave it at the basic size, definitely not make it smaller than 16px